lib/vparse.h

```diff
@@ -5,6 +5,8 @@
 #include "util.h"
 #include "strarray.h"
 
+#define APPLE_LABEL_PROPERTY "X-ABLabel"
```

Comment:
I'd prefix that name with either `VCARD_` or `VPARSE_`.

Reply:
Done